Nosenti/Innocent-Ingabire-portfolio
# Ft create query validation

#### What does this PR do?
 validate query and blog endpoints
#### Description of Task to be completed
validated query and blog endpoints
#### How should this be manually tested?
clone the repository, cd inside the repository and run the command npm start
Use postman to test query and blog endpoints with the wrong data and you should get customized errors
#### Any background context you want to provide?
#### What are the relevant stories?
<blockquote class="trello-card"><a href="https://trello.com/c/YOwyb2fB/34-validate-enpoints">validate enpoints</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>
